src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -45,6 +46,16 @@ export default function ConfirmedAppointmentListItem({
     },
   );
 
+  let appointmentState;
+
+  if (canceled) {
+    appointmentState = 'Canceled';
+  } else if (isPastAppointment) {
+    appointmentState = 'Completed';
```

Comment:
I don't know that we want to say completed here. We don't know for sure if the person made it to the appointment; apparently this info is always accurately recorded in VistA. I think maybe the designs left this off entirely? Not sure.